### PR TITLE
[STORM-2365] Support for specifying output stream in event hubs spout

### PR DESCRIPTION
**Pull request by arunmahadevan**

### CODE REVIEW

**Comment by satishd** [comment]
+1 LGTM

**Comment by HeartSaVioR** [comment]
+1

**Comment by harshach** [comment]
@HeartSaVioR did this get into 1.x-branch as well?

**Comment by satishd** [comment]
@harshach This change was cherrypicked and merged earlier into 1.x-branch.

**Comment by HeartSaVioR** [comment]
@satishd Please mark JIRA issue resolved with proper fix versions. The CHANGELOG is easy to be broken (this issue is marked as 2.0.0 from CHANGELOG) so we shouldn't rely on that.

**Comment by HeartSaVioR** [comment]
(I saw 1.x-branch has CHANGELOG containing STORM-2365 in 1.1.0... It should be hard to follow, and I feel maintaining CHANGELOG for branches are useless.)
